app/src/main/kotlin/com/wire/android/ui/authentication/devices/register/RegisterDeviceScreen.kt

```diff
@@ -76,7 +79,9 @@ fun RegisterDeviceScreen() {
         onContinuePressed = viewModel::onContinue,
         onErrorDismiss = viewModel::onErrorDismiss,
         onBackButtonClicked = clearSessionViewModel::onBackButtonClicked,
-        onCancelLoginClicked = clearSessionViewModel::onCancelLoginClicked,
+        onCancelLoginClicked = remember(clearSessionViewModel, navigator) {
+            { clearSessionViewModel.onCancelLoginClicked(NavigationSwitchAccountActions(navigator::navigate)) }
+        },
```

Comment:
Why `remember` is used here ?

Reply:
Generally lambdas are usually unstable, especially when they use things like ViewModel, so they will make it that the child composable will always be recomposed, because a new lambda is created every time so the recomposer cannot skip that one, there's a great article explaining this here: 
https://multithreaded.stitchfix.com/blog/2022/08/05/jetpack-compose-recomposition/

There are some solutions to skip recomposition if not necessary, one of them is to use references which we try to use as much as possible but here it's not possible, but we can also remember lambdas and it will also do the trick.

In this case, however, I just wanted to be extra safe, but there's probably no need, because we have only a single child which will be recomposed if anything changes anyway, so I removed `remember` from these screens.